### PR TITLE
Show and Hide URP Form from admin page

### PR DESCRIPTION
**Pull request by leej42**
Create an API call for displayURP and getDisplayURP
Called the API in the navbar to show/hide the URP Form
Added a field to the classyear Model
Added user=admin to delete class year data

### CODE REVIEW

**Comment by leej42** [comment]
fixes #139

File: client/components/navbar/navbar.controller.js [hunk 1]
```diff
@@ -27,4 +27,17 @@ angular.module('observatory3App')
...
       return route === $location.path();
     };
+
+    // Toggles the display of URP form
+    var URPDisplay = function(){
+      $http.get("/api/classyear/displayURP")
+        .success(function(data){
+          $scope.displayURP = data.displayURP;
+        });
+        console.log("did it");
```
Comment (agundy) [inline client/components/navbar/navbar.controller.js]: We should remove these console.log lines

**Comment by agundy** [comment]
This does not fail correctly if there is no class year defined. You should add in a default if the lookup fails. Shouldn't make a difference in production but matters for development.

**Comment by seveibar** [comment]
class year's will never not be defined- correction is a low priority issue

**Comment by seveibar** [comment]
create #168 to address the issue, merging

**Pull request merged**